aws_sqs/lib/botocore/data/fsx/2018-03-01/examples-1.json

```diff
@@ -0,0 +1,438 @@
+        },
+        "description": "This operation creates a new backup.",
+        "id": "to-create-a-new-backup-1481840798597",
+        "title": "To create a new backup"
+      }
+    ],
+    "CreateFileSystem": [
+      {
+        "input": {
+          "ClientRequestToken": "a8ca07e4-61ec-4399-99f4-19853801bcd5",
+          "FileSystemType": "WINDOWS",
+          "KmsKeyId": "arn:aws:kms:us-east-1:012345678912:key/1111abcd-2222-3333-4444-55556666eeff",
+          "SecurityGroupIds": [
+            "sg-edcd9784"
+          ],
+          "StorageCapacity": 3200,
+          "StorageType": "HDD",
+          "SubnetIds": [
+            "subnet-1234abcd"
+          ],
+          "Tags": [
+            {
+              "Key": "Name",
+              "Value": "MyFileSystem"
+            }
+          ],
+          "WindowsConfiguration": {
+            "ActiveDirectoryId": "d-1234abcd12",
+            "Aliases": [
+              "accounting.corp.example.com"
+            ],
+            "AutomaticBackupRetentionDays": 30,
+            "DailyAutomaticBackupStartTime": "05:00",
+            "ThroughputCapacity": 32,
+            "WeeklyMaintenanceStartTime": "1:05:00"
+          }
+        },
+        "output": {
+          "FileSystem": {
+            "CreationTime": "1481841524.0",
+            "DNSName": "fs-0123456789abcdef0.fsx.com",
+            "FileSystemId": "fs-0123456789abcdef0",
+            "KmsKeyId": "arn:aws:kms:us-east-1:012345678912:key/1111abcd-2222-3333-4444-55556666eeff",
+            "Lifecycle": "CREATING",
+            "OwnerId": "012345678912",
+            "ResourceARN": "arn:aws:fsx:us-east-1:012345678912:file-system/fs-0123456789abcdef0",
+            "StorageCapacity": 3200,
+            "StorageType": "HDD",
+            "SubnetIds": [
+              "subnet-1234abcd"
+            ],
+            "Tags": [
+              {
+                "Key": "Name",
+                "Value": "MyFileSystem"
+              }
+            ],
+            "VpcId": "vpc-ab1234cd",
+            "WindowsConfiguration": {
+              "ActiveDirectoryId": "d-1234abcd12",
+              "Aliases": [
+                {
+                  "Lifecycle": "CREATING",
+                  "Name": "accounting.corp.example.com"
+                }
+              ],
+              "AutomaticBackupRetentionDays": 30,
+              "DailyAutomaticBackupStartTime": "05:00",
+              "ThroughputCapacity": 32,
+              "WeeklyMaintenanceStartTime": "1:05:00"
+            }
+          }
+        },
+        "comments": {
+        },
+        "description": "This operation creates a new Amazon FSx for Windows File Server file system.",
+        "id": "to-create-a-new-file-system-1481840798547",
+        "title": "To create a new file system"
+      }
+    ],
+    "CreateFileSystemFromBackup": [
+      {
+        "input": {
+          "BackupId": "backup-03e3c82e0183b7b6b",
+          "ClientRequestToken": "f4c94ed7-238d-4c46-93db-48cd62ec33b7",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical
<details>
<summary><strong>Description</strong></summary>

A generic secret or password is an authentication token used to access a computer or application and is assigned to a password variable.

</details>


Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 4319f63b-8020-44e3-ae15-9d78cc59f042 -->